src/model/page-config.ts

```diff
@@ -14,14 +14,13 @@
  * limitations under the License.
  */
 
-/**
- */
 export class PageConfig {
-  /**
-   * @param {string} productOrPublicationId
-   * @param {boolean} locked
-   */
-  constructor(productOrPublicationId, locked) {
+  publicationId_: string;
+  productId_: string | null;
+  label_: string | null;
+  locked_: boolean;
```

Comment:
Looks like these should be private, and as noted in the other PR we can optionally remove the underscore from the names.

Reply:
For now, the `_` suffixes are improving minification